aten/src/ATen/core/dispatch/DispatchKeyExtractor.h

```diff
@@ -25,7 +25,7 @@ const DispatchKeySet always_included{DispatchKey::BackendSelect};
 //
 // Unlike Tensor::key_set(), the value of this on a tensor can change depending
 // on TLS.
-static inline DispatchKey dispatchTypeId(
+static inline DispatchKeySet dispatchTypeId(
```

Comment:
Hmm...

Reply:
Yeah.... See the reply I put on your comment on `callWithDispatchKeySet()` - I no longer want to return just the highest key in the set, now that we want to save the entire set for plumbing purposes. But this function should probably be renamed. `computeDispatchKeySet()`?